test/test_images/pubsub_target/main.go

```diff
@@ -0,0 +1,102 @@
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+	"strconv"
+	"time"
+
+	cloudevents "github.com/cloudevents/sdk-go"
+	"github.com/google/knative-gcp/test/e2e/lib"
+	"github.com/kelseyhightower/envconfig"
+)
+
+func main() {
```

Comment:
This part doesn't use the test framework that other xxx_target use. Will change it later